Source/SwiftyJSON.swift

```diff
@@ -131,6 +131,22 @@ public struct JSON {
         }
     }
 
+    /**
+     Parses the object confirming to Encodable protocol into a JSON object using JSONEncoder
```

Comment:
I assume you meant _conforming_ here slight keystroke mixup since "i" and "o" are next to each other 😉